assets/css/global.less

```diff
@@ -1970,6 +1970,13 @@ body .guides .guides-section-white {
   }
   h1, h2, h3, h4, h5, h6 {
     color: @guides-font;
+    margin: 10px 0; 
```

Comment:
Why add a margin on top and bottom?

Reply:
The mockup has some margin for all header tags.

Reply:
Yes, but I believe the spacing for headers looks correct already in master, after a bunch of tweaks from yesterday. Can you show what change this is making?